html/home.html

```diff
@@ -0,0 +1,61 @@
+<html>
+    <head> 
+        <meta charset="utf-8">
+        <title>Yellow Brick Code</title>
+        <link rel="stylesheet" href="style.css">
+        <!-- BootStrap JavaScript imports -->
+        <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+        <!-- BootStrap CSS imports -->
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+        <link rel="stylesheet" href="lib/codemirror/lib/codemirror.css">
+    </head>
+    <body>
+        <!--<nav>
+            THE ORIGINAL NAV BAR CODE
+            <ul>
+                <li><a href ="home.html">Home</a></li>
+                <li><a href ="sign-in.html">Sign-in</a></li>
+                <li><a href ="about.html">About Us</a></li>
+                <li><a href ="contact.html">Contact</a></li>
+            </ul>
+        </nav>-->
+        
+        <nav class="navbar navbar-expand-lg navbar-light bg-light">
+            <a class="navbar-brand" href="#">Yellow Brick Code</a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                <ul class="navbar-nav mr-auto">
+                    <li class="nav-item active">
+                        <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="sign-in.html">Sign-in</a>
```

Comment:
this link isn't working for me, i think you need to change to the route to /sign-in (also, i got an authorization error when i tried signing in)

Reply:
This is what I need to talk with Ian about because there are some credential set ups that need to happen